src/pages/blogpage/BlogPage.jsx

```diff
@@ -0,0 +1,66 @@
+import {Link, useNavigate, useParams} from "react-router-dom";
+
+import './BlogPage.css'
+import TimeDate from '../../helpers/TimeDate.js'
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {CaretLeft, Clock} from "@phosphor-icons/react";
+
+
+
+function BlogPage() {
+    const { id } = useParams();
+
+
+    const [oneBlog,setOneBlog]=useState([])
```

Comment:
`oneBlog` zal nooit een array zijn, dus dan zou ik ook geen default waarde als een lege array pakken. Ik zou zelf gaan  voor `undefined`, omdat je check of het dan al is ingeladen wat makkelijker wordt
```suggestion
    const [oneBlog,setOneBlog]=useState();
```